Update screening and eligibility requirements for `Official tournament support` (#8754)
ppy/osu-wiki: wiki/Tournaments/Official_support/en.md

Once screening concludes, the account support team will provide a list of any users who failed screening and are not considered eligible for tournament play, without providing specific reasoning. Individual users who are unhappy with their screening outcome should be told to consult [accounts@ppy.sh](mailto:accounts@ppy.sh) via email.

> Only players who are participating in the tournament or would be likely substitutes for a team made during the course of the tournament should be sent for screening. For example, in a country-based tournament using a tryouts system, organizers are encouraged to send only the players determined to be participating after the tryouts process concludes, along with several substitutes for each team in the event one or more of the players on a given country's team get screened. For clarifications on whether a tournament should send all registrations or only relevant registrations, please contact a member of the [Tournament Committee](/wiki/People/Tournament_Committee).

Reviewer: part of the first sentence i believe is redundant. i'm not sure if there's an intentional distinction to make here (or whether it makes sense).

last sentence sounds a bit baffling. if i'm off the mark please explain.

```suggestion
Only players who are participating in the tournament or would be likely substitutes for a team should be sent for screening. For example, in a country-based tournament using a tryouts system, organisers are encouraged to send only the players determined to be participating after the tryouts conclude, along with several substitutes for each team in the event they need to participate. For clarifications on which registrants are relevant for screening for a particular tournament, contact the [Tournament Committee](/wiki/People/Tournament_Committee).
```

Author: The first sentence is not redundant. This change is specifically for tournaments where completed rosters are not submitted with registrations. Tournament Committee does not want this change to apply to all team-based tournaments, only to those where rosters are formed and submitted for screening during the course of the tournament.

All other changes are good.
